iquip/explorer.py

```diff
@@ -0,0 +1,33 @@
+"""
+App module for showing the experiment list and opening an experiment.
+"""
+
+from typing import Optional, Tuple
+
+from PyQt5.QtCore import QObject
+from PyQt5.QtWidgets import QLabel, QVBoxLayout, QWidget
+
+from qiwis import BaseApp
```

Comment:
Importing `qiwis` flatly would be better, because we might want to use other functions or classes in it, e.g., `loads()`, `Serializable`, etc.